docs/getting-started/quickstart.md

```diff
@@ -68,19 +79,19 @@ External programs talk to Redis using a TCP socket and a Redis specific protocol
 
 The first thing to do in order to check if Redis is working properly is sending a **PING** command using redis-cli:
 
-    $ redis-cli ping
+    $ redis-cli PING
```

Comment:
```suggestion
    $ redis-cli ping
```

Reply:
@justincastilla Since commands aren't case-sensitive, I think we should leave as is.